var/spack/repos/builtin/packages/py-pyside/package.py

```diff
@@ -38,6 +38,7 @@ class PyPyside(Package):
 
     extends('python')
     depends_on('py-setuptools', type='build')
+    depends_on('py-sphinx')
```

Comment:
Why is Sphinx need to build Python bindings for Qt4?  I assume this is for PySide documentation?  Can you make it optional, under a variant called `+doc`?